### DISCUSSION TITLE
SEO Improvements

### DISCUSSION THREAD

**Discussion by giliomeejg** [gitcoinco/grants-stack#1199] [General]
Here are a couple of thoughts to improve the SEO of some of our Explorer pages.

- Update the <title> with something specific for the round page, e.g. for [https://grant-explorer.gitcoin.co/#/round/1/0x...938](https://grant-explorer.gitcoin.co/#/round/1/0xdf75054cd67217aee44b4f9e4ebc651c00330938), the title could be 'UNICEF Round on Gitcoin'
- Update the <title> of a specific project page, e.g. for [https://grant-explorer.gitcoin.co/#/round/1/0xdf...938](https://grant-explorer.gitcoin.co/#/round/1/0xdf75054cd67217aee44b4f9e4ebc651c00330938/0x30be8ac1c84158cbc78c55bbffda007a3d3ea4cd2b10afd154279da82ee35b4f-0xdf75054cd67217aee44b4f9e4ebc651c00330938) this could be 'Pixframe Studios - Transforming Education on gitcoin'
- Update the <meta> descriptions to use the first paragraph of the content on the page.
- Consider a [URL structure that reduces the use of URL fragments](https://github.com/gitcoinco/grants-round/discussions/969)

**Comment by vacekj** [comment]
@giliomeejg thank you for the suggestions - some of those are currently blocked on us using CRA on all apps. I have an RFC to move to Next on explorer in the works, so this will be fixed when that happens.
